### ISSUE TITLE
How to tell who disconnect ws?

### ISSUE THREAD

**Issue by poor1017** [uNetworking/uWebSockets#1642]
Hi @uNetworkingAB 

Thank you for your project!

I want to know how to tell who disconnect ws in `.close`? Server or client?

In `.close = [this](auto *ws, int code, std::string_view message) {`
I can only get `code`, but the value `1000` can not identify the one who disconnect.

Thank you.

**Comment by poor1017** [comment]
Thank you for your reply!

It means if I got a `1000`, then the connection is closed by client?

Thank you!

**Comment by uNetworkingAB** [comment]
Yes and you can also see this if you look at WireShark or similar tcpdump

**Comment by poor1017** [comment]
Thank you!

**Comment by poor1017** [comment]
Hi @uNetworkingAB 

I got a `1006`, maybe this is caused by fire wall or VPN.

How to prove this?(ws is closed by client side fire wall or VPN)

Thank you!
